examples/web-sdk-sample/packages/vue2/src/App.vue

```diff
@@ -37,6 +37,14 @@
             >Websocket</router-link
           >
         </span>
+        <span class="nav-item">
+          <router-link
+            to="/upload-file"
+            class="nav-link"
+            :class="{ 'active-link': isActive('/upload-file') }"
+          >UploadFile</router-link
+          >
```

Comment:
No format ? use **prettier** to format these code. 
.vscode/settings.json: (examples/web-sdk-sample) folder
```
{
  "editor.codeActionsOnSave": {
    "source.fixAll.eslint": "explicit"
  },
  "editor.formatOnSave": true, // Prettier auto format on save
  "editor.defaultFormatter": "esbenp.prettier-vscode"
}
```

Reply:
Reformat all the new file with prettier in WebStorm